BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -322,9 +322,11 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     sender.loaderView = LoaderView(size: .small)
     sender.loaderPlacement = .replacesContent
     sender.isLoading = true
+    sender.isEnabled = false
     ledgerObserver.grantClaimed = { [weak self] grant in
       guard let self = self, let grantAmount = BATValue(probi: grant.probi)?.displayString else { return }
       sender.isLoading = false
+      sender.isEnabled = true
```

Comment:
This is possibly not needed since the button goes away, but doesn't hurt to add it here imo